roles/repo-setup/tasks/repo-Debian.yaml

```diff
@@ -0,0 +1,12 @@
+---
+
+- name: Setup newrelic infra APT key
+  apt_key:
+    url: "{{ gpg_key }}"
+
+- name: Setup APT repository
+  apt_repository:
+    repo: "deb {{ repo_endpoint }}/linux/apt {{ ansible_distribution_release }} main"
+    filename: newrelic-infra
+
```

Comment:
shouldn't we also run apt-cache update like in ubuntu one?
```
- name: Run the equivalent of "apt-get update" as a separate step
  apt:
    update_cache: yes
  ignore_errors: true #ubuntu 20.10 image has broken repos so ignore errors
```
both share package-debian and it has `update_cache=no`